src/cascadia/TerminalSettingsEditor/Converters.idl

```diff
@@ -14,6 +14,11 @@ namespace Microsoft.Terminal.Settings.Editor
         FontWeightConverter();
     };
 
+    runtimeclass InvertedBooleanConverter : [default] Windows.UI.Xaml.Data.IValueConverter

```

Comment:
how many of these freaking converters do we need lol

Reply:
_yeaaaah_. This is just...dumb... We could make this all a part of the view model. But like, this is so much more straightforward and I have a feeling we'll need it again at some point haha

Reply:
We have introduced another whole converter _and refactored the converter code_ just for this highly controversial string.